lib/rucio/db/sqla/models.py

```diff
@@ -61,6 +61,15 @@ def compile_binary_oracle(type_, compiler, **kw):
     return "NUMBER(1)"
 
 
+@event.listens_for(Table, "before_create")
```

Comment:
This is problematic like this, as it makes the models.py super hard to read. Doesn't it work if the string below is just
`"'GROUPING' IS NOT NULL"` ?

Reply:
`"'GROUPING' IS NOT NULL"` seems not to work on mysql and ``"`GROUPING` IS NOT NULL"`` does not work on postgres unfortunately

Reply:
`CheckConstraint(grouping!=None, name='RULES_GROUPING_NN'),` should give the dialect appropriate identifier quoting.

Reply:
Yes that worked, thanks!